Resources/views/CRUD/list_outer_rows_list.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 
-{% for object in admin.datagrid.results %}
+{% for object in datagrid_results %}
```

Comment:
Are you sure that the ``datagrid_result`` exists here? You could override ``base_list.html.twig`` without definging ``datagrid_result``.